packages/frontend/app/PublicEnv.tsx

```diff
@@ -0,0 +1,23 @@
+export interface PublicEnvironment {
+  OPEN_PAYMENTS_URL: string
+}
+
+declare global {
+  interface Window {
+    ENV: PublicEnvironment
+  }
+}
+
+type PublicEnvProps = {
+  env: PublicEnvironment
+}
+
+export const PublicEnv = ({ env }: PublicEnvProps) => {
+  return (
+    <script
+      dangerouslySetInnerHTML={{
+        __html: `window.ENV = ${JSON.stringify(env)}`
+      }}
+    />
+  )
+}
```

Comment:
Resolved [the original comment](https://github.com/interledger/rafiki/pull/1621/files/0f32cd386da9e7be4d9df3602afbe139e857aee0#r1276524828) because I complete failed at highlighting the code :laughing:

Is the `OPEN_PAYMENTS_URL` actually getting used anywhere? I see that we're making it available in the browser but it seems like its unused?

Reply:
It's used as add-on content for this input (when creating a payment pointer).
https://github.com/interledger/rafiki/blob/0f32cd386da9e7be4d9df3602afbe139e857aee0/packages/frontend/app/routes/payment-pointers.create.tsx#L50-L57